src/rtc/geogrid.py

```diff
@@ -268,15 +273,58 @@ def get_point_epsg(lat, lon):
         raise ValueError(err_str)
 
 
+def _check_pixel_spacing(y_spacing_positive, x_spacing, epsg, product_str):
+    '''
+    Verify pixel spacing
 
+    Parameters
+    ----------
+    y_spacing_positive: float
+        Y/latitude pixel spacing (absolute value)
+    x_spacing: float
+        X/longitude pixel spacing
+    epsg: int
+        EPSG code
+    product_str: string
+        Product type string (e.g., "Mosaic" or "Bursts")
 
+    Returns
+    -------
+    y_spacing: float
+        Y/latitude pixel spacing
+    x_spacing: float
+        X/longitude pixel spacing
+    '''
+    # Check spacing in X direction
+    if x_spacing is not None and x_spacing <= 0:
+        err_str = f'{product_str} pixel spacing in X/longitude direction must'
+        err_str += f' be positive (x_spacing: {x_spacing})'
+        logger.error(err_str)
+        raise ValueError(err_str)
+    elif x_spacing is None and epsg and epsg == 4326:
+        x_spacing = SECONDS_IN_DEG
+    elif x_spacing is None and epsg:
+        x_spacing = 30
 
+    # Check spacing in Y direction
+    if y_spacing_positive is not None and y_spacing_positive <= 0:
+        err_str = f'{product_str} pixel spacing in Y/latitude direction must'
+        err_str += f' be positive (y_spacing: {y_spacing_positive})'
+        logger.error(err_str)
+        raise ValueError(err_str)
+    elif y_spacing_positive is None and epsg and epsg == 4326:
+        y_spacing = -SECONDS_IN_DEG
+    elif y_spacing_positive is None and epsg:
+        y_spacing = -30
+    elif y_spacing_positive is not None:
+        y_spacing = - y_spacing_positive
```

Comment:
```suggestion
        y_spacing = -y_spacing_positive
```
Nit-pick here. PEP8 does not seem to mention about the spacing between the negation in unary operation, but I think it would be good to keep using the one style.
More information about what other language says about it:
https://softwareengineering.stackexchange.com/questions/189053/space-between-negative-sign-and-variable-name